app/views/surveys/_organisational_fields.html.erb

```diff
@@ -285,7 +285,7 @@
 
   <p>
     Organisation funding sources:
-    <% sources = survey.settings["organisation_funding_sources"].map{|k| surveys_data("organisation_funding_sources")[k.to_sym]}.compact! %>
```

Comment:
Could you explain why did you remove the `to_sym` bit, it was working well for me before I think.

Reply:
Because it wasn't working for me... It was returning nil.